### PR TITLE
Add Friday Night Funkin' Resources

### PR DESCRIPTION
**Pull request by lime360**
**[add this list](https://github.com/FunkinCrew/funkin-resources)**

this list is all about fnf and i was wondering if i can put it here lol

this list is made by @FunkinCrew (fnf developers), not me

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by lime360** [comment]
I DIDN'T MADE IT, @FunkinCrew MADE THIS LIST AAAAAAAAAAAAAAAAAA!

nobody submitted it so i should do it

**Comment by vhladiienko** [comment]
Hi @lime360 

I think your PR misses some love:

1. Link other PRs in this repo, that you have reviewed or commented on
2. Your list entry should be at the bottom of the category you're updating

As for the list itself, it needs some adjustments too.

1. First of all, make sure it passes linting. This will cover most of the issues with the cuurent revision.
2. "Further reading" section should be moved to the Footnotes section.

Even though you're not the author of the list, I'm sure you can contribute to make the list comply with the requirements.

**Comment by Karneades** [comment]
@lime360 I understood that someone else created that list. To support Sindre a bit, I comment on this issue.

As Sindre noted above, for a list to be included in the awesome-list, that list should fulfill various requirements. When looking at the repo, many things from the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) are still to be improved / changed. @FunkinCrew must change some of the project things and the list itself must be improved according to the guidelines.

Hope you keep working on the list that it get's merged. For new lists there is also https://github.com/sindresorhus/awesome/issues/2242 which would be the place for such a list in "draft" mode for the future.

**Comment by Superlegend717** [comment]
Hello sir,iam a learner i need to know how can i operate termux in an android,please!

**Comment by lime360** [comment]
@Superlegend717 i am sorry but this is off-topic.

**Comment by Superlegend717** [comment]
Ok,thank you so much for respond.

**Comment by lime360** [comment]
i edited my first pr message

**Comment by EliteMasterEric** [comment]
Hey,

I am a maintainer for the Friday Night Funkin' Resources repository. I will be looking through this thread and attempting to fix the issues I can. Feel free to @ me if you have any other concerns.

File: readme.md [hunk 1]
```diff
@@ -547,6 +547,7 @@
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
 - [Game Datasets](https://github.com/leomaurodesenv/game-datasets#readme) - Materials and datasets for Artificial Intelligence in games.
 - [Haxe Game Development](https://github.com/Dvergar/awesome-haxe-gamedev#readme) - A high-level strongly typed programming language used to produce cross-platform native code.
+- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources#readme) - A rhythm game made in HaxeFlixel.
```
Comment (EliteMasterEric) [inline readme.md:550]: I would change this line to read the following:

```
- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources) - A cartoon rhythm game made for Newgrounds.
```

**Review by EliteMasterEric — CHANGES_REQUESTED** [review]

**Comment by EliteMasterEric** [comment]
As of time of writing, the only pending issue I am aware of on the Funkin' Resources repository is the age one. All other linting issues have been resolved.

Please feel free to provide any additional feedback or PRs if necessary.

File: readme.md (hunk 1 above)
Comment (Karneades) [reply readme.md:550]: According to the guidelines, the #readme is required which is also more user friendly to directly jump to it when opening the link.

**Comment by Karneades** [comment]
> As of time of writing, the only pending issue I am aware of on the Funkin' Resources repository is the age one. All other linting issues have been resolved.
> 
> Please feel free to provide any additional feedback or PRs if necessary.

Nice improvements since last time I looked over it 2 weeks ago. The repo name as required by the guidelines needs an "awesome-..." prefix.
* See part around _The repo name of your list should be in lowercase slug format: awesome-name-of-list._

**Comment by EliteMasterEric** [comment]
> The repo name as required by the guidelines needs an "awesome-..." prefix.
> 
>     * See part around _The repo name of your list should be in lowercase slug format: awesome-name-of-list._

Ah, I misunderstood this, I interpreted that statement as "the repo must be in lower kebab case" and not "the repo must be in lower kebab case and start with `awesome`-".

I will look into getting it changed.

**Comment by EliteMasterEric** [comment]
At time of writing, **ALL KNOWN ISSUES ARE CURRENTLY FIXED**, including the one regarding repo age.

The repository now passes all linting checks.

**Comment by EliteMasterEric** [comment]
The workflow will need to be re-run in order for GitHub to validate the changes.

File: readme.md (hunk 1 above)
Comment (EliteMasterEric) [reply readme.md:550]: Ok so the correct description should be:

```
- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin-resources#readme) - A cartoon rhythm game made for Newgrounds.
```

**Review by EliteMasterEric — CHANGES_REQUESTED** [review]

**Review by eric-hc — CHANGES_REQUESTED** [review]
The repo name should be prefixed with `awesome-`

image

**Comment by jacobdjwilson** [comment]
It may make sense to use Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for further definition of this list.

**Comment by EliteMasterEric** [comment]
> It may make sense to use Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for further definition of this list.

Incorrect. This list is only pending the resolution of minor fixes before being reapproved for submission.

**Comment by ElizaLexi** [comment]
I really enjoyed reading the article "**Why Has Friday Night Funkin' Become So Popular**" on Medium. It provided valuable insights into the game's rising popularity. The writer did a fantastic job capturing the essence of the game and its impact on the gaming community. However, I suggest including more specific examples and statistics to further support the claims made. Overall, a great read! For more engaging content, visit **WWW.APPZSOFT.COM.**

**Comment by sindresorhus** [comment]
The readme header should be `Awesome Friday Night Funkin'`, not `Friday Night Funkin' Resources`.

**Pull request closed without merging**